examples/multi_cn/s0/local/word_segment.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python2.7
+#coding:utf-8
```

Comment:
what's the difference to word_segementation.py?

Reply:
word_segment.py use mmseg with python2.7, word_segmentation.py use jieba with python2 or python3

Reply:
I see, word segment is not required for e2e, better to remove it?

Reply:
Got it